access-grant/src/main/java/com/inrupt/client/accessgrant/AccessGrantClient.java

```diff
@@ -922,7 +922,7 @@ static boolean isAccessDenial(final URI type) {
     /**
      * A data object for verification responses.
      */
-    static class VerificationResponseData {
+    static public class VerificationResponseData {
```

Comment:
Does this need to be public? It was intentionally non-public

Reply:
That's my understanding of the following error: 
```
Caused by: jakarta.json.bind.JsonbException: Error accessing field 'checks' declared in 'class com.inrupt.client.accessgrant.AccessGrantClient$VerificationResponseData'
...
Caused by: java.lang.IllegalAccessException: class is not public: com.inrupt.client.accessgrant.AccessGrantClient$VerificationResponseData.checks/java.util.List/getField, from public Lookup
```
The error is not thrown anymore after the class is made public.

Reply:
Instead of making this inner class public, I would be more inclined to remove it and make the existing publicly facing class mutable

Reply:
Fixed in 23cc8dec835cf9b60062ce5c41bcdfbdcaa9f5ef